integration_test/info_test.go

```diff
@@ -37,3 +46,22 @@ func TestKrewInfoInvalidPlugin(t *testing.T) {
 		t.Errorf("Expected `krew info %s` to fail", plugin)
 	}
 }
+
+func TestKrewInfoCustomIndex(t *testing.T) {
+	skipShort(t)
+
+	test, cleanup := NewTest(t)
+	defer cleanup()
+
+	test = test.WithEnv(constants.EnableMultiIndexSwitch, 1).WithIndex()
+	defaultIndexRoot := test.TempDir().Path("index/" + constants.DefaultIndexName)
+
+	test.Krew("index", "add", "foo", defaultIndexRoot).RunOrFail()
```

Comment:
That trick has been used quite often and it's not straightforward what's happening. Mind to extract this into a helper?

Reply:
I will open an issue to do this later. This multi-index work has caused few other cleanup items that are increasing visible.

Reply:
opened #565